controllers/controllers.js

```diff
@@ -0,0 +1,98 @@
+const { HttpError } = require("../helpers");
+const Joi = require("joi");
+const {
+  listContacts,
+  getContactById,
+  addContact,
+  updateContact,
+  removeContact,
+} = require("../models/contacts");
+
+const contactSchema = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().email().required(),
```

Comment:
schema should be in separate file. You should validate email and phone properly

Reply:
fixed